rmgpy/pdep/cse.pyx

```diff
@@ -223,3 +223,163 @@ def apply_chemically_significant_eigenvalues_method(network, list lumping_order=
 
     # Return the matrix of k(T,P) values and the pseudo-steady population distributions
     return k, pa
+
+def get_rate_coefficients_CSE_Advanced(network,T,P,neglect_high_energy_collisions=False,high_energy_rate_tol=0.01):
+    """
+    CSE using the Georgievskii et al. 2013 method
```

Comment:
Maybe put the full reference here for future developers

Reply:
added